orttraining/orttraining/training_ops/cpu/torch/torch_custom_function_kernel_base.cc

```diff
@@ -164,11 +164,8 @@ void PythonOpBase::CreateArgPositions() {
   }
 }
 
-void PythonOpBase::SetContextOutput(OpKernelContext* context, std::vector<void*>& returned_args) const {
-  // Handle the outputs;
-  // The 1st output is context index of auto grad function.
-  // Other outputs are address of OrtValue we got from python script run.
-  PyObject* ctx_addr = reinterpret_cast<PyObject*>(returned_args[0]);
+void PythonOpBase::SetContextOutput(OpKernelContext* context, std::vector<void*>& returned_raw_pointers) const {
```

Comment:
Maybe let's focus on "single pointer" case, since the only raw pointer passed in is the `context` object. I will be change the `vector<void*>` to `void*`.